### ISSUE TITLE
Postgres QueryInterface.removeIndex() problem with schemas

### ISSUE THREAD

**Issue by Etzos** [sequelize/sequelize#5642]
When attempting to remove an index from a table that has a (Postgres-type) schema, the call errors out with the following message:

```
Unhandled rejection TypeError: identifiers.indexOf is not a function
    at Object.QueryGenerator.quoteIdentifiers (.../sequelize/lib/dialects/abstract/query-generator.js:943:21)
    at Object.QueryGenerator.removeIndexQuery (.../sequelize/lib/dialects/postgres/query-generator.js:414:23)
    at QueryInterface.removeIndex (.../sequelize/lib/query-interface.js:485:33)
[snip]
```

A quick glance through the code shows that the `addIndexQuery()` method handles tables with schemas attached properly [1] however the Postgres version of `removeIndexQuery()` does not [2].

**Version:** Current (3.20.0), problem is visible in current master.

[1] https://github.com/sequelize/sequelize/blob/7764212aab675b9b0278e80f3c18660e766cb197/lib/dialects/abstract/query-generator.js#L675
[2] https://github.com/sequelize/sequelize/blob/7764212aab675b9b0278e80f3c18660e766cb197/lib/dialects/postgres/query-generator.js#L413

**Comment by mickhansen** [comment]
The handling in addIndexQuery is odd, it should actually be removed. Are you passing tablename as a string ala `schema.table`?

The idea is support a format ala `{schema: 'schema', table: 'table'}` but perhaps we could introduce a parser that turns a delimited string into that.

**Comment by Etzos** [comment]
Neither one works if I remember correctly, I know that I pass everything as `{ schema: 'schema', table: 'table' }` so I am positive I tested that. I'm not entirely sure if I tested using a string, but I don't think that would work either.

I realize that the handling in `addIndexQuery()` is a bit odd as it has special handling that is obviated by the use of `quoteTable()`, my point was that `removeIndexQuery()` should probably be using `this.quoteTable(tableName)` instead since that knows how to handle schemas from the look of it.
